test cases/frameworks/7 gnome/resources-data/meson.build

```diff
@@ -0,0 +1,14 @@
+subdir('subdir')
+
+script = '''
+import os, sys
+assert os.path.exists(sys.argv[1]), "File %s not found" % sys.argv[1]
+print("Unimportant generated content")
+'''
+
+res3_txt = custom_target('res3.txt',
+  input: 'res3.txt.in',
+  output: 'res3.txt',
+  command: ['python', '-c', script, '@INPUT@'],
+  capture: true,
+)
```

Comment:
Actually it didn't create the file because I hadn't run ninja, only meson :/ ... so 'capture' does indeed work as expected